app/code/community/Lesti/Fpc/Helper/Data.php

```diff
@@ -258,4 +259,30 @@ public function getContentType(\Mage_Core_Controller_Response_Http $response)
 
         return 'text/html; charset=UTF-8';
     }
+
+    /**
+     * Transform array parameter to string to avoid error and make correct validation in regexp pattern
+     *
+     * @param mixed $param Input parameter
+     *
+     * @return mixed
+     */
+    protected function _paramToString($param)
+    {
+        if (is_array($param)) {
+            $param = array_reduce(
+                $param,
+                function ($carry, $item) {
```

Comment:
With conflicts I mean something like this:
* We want to ignore requests where multiple filters for an attribute contain only the value `25`. So we add the to the `miss_uri_params` something like this `attribute_code=/^25$/` (_I'm not 100% sure, but I guess that should work_).
* A request with the multiple filters for the attribute contains the values `2` and `5`. The items will be concat to `25`  and the request gets ignored.
* [json_encode](http://php.net/manual/en/function.json-encode.php) was just my quick idea, cause it would be possible to create a regular expression (_maybe a complicated one_) that only covers the example from above without ignoring other requests.

I really appreciate your PR, but the issue https://github.com/GordonLesti/Lesti_Fpc/issues/191 is so old cause I haven't found a good solution that covers every case and that ignoring with regular expressions is comfortable at the same time.